mu-plugins/central-hub/tests/phpunit/integration/meta-data/getMetaBoxKeys.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Tests for the function get_meta_box_keys().
+ *
+ * @package     spiralWebDb\centralHub\Tests\Integration\Metadata
+ * @since       1.3.0
+ * @author      Robert Gadon <rgadon107>
+ * @link        https://github.com/rgadon107/cornerstone
+ * @license     GNU-2.0+
+ */
+
+namespace spiralWebDb\centralHub\Tests\Integration\Metadata;
+
+use function KnowTheCode\ConfigStore\loadConfig;
+use function spiralWebDB\Metadata\get_meta_box_keys;
+use spiralWebDb\Cornerstone\Tests\Integration\Test_Case;
+
+/**
+ * Class Tests_GetMetaBoxKeys
+ *
+ * @package spiralWebDb\centralHub\Tests\Integration\Metadata
+ * @group   meta-data
+ */
+class Tests_GetMetaBoxKeys extends Test_Case {
+
+	/**
+	 * Empty the store before starting these tests.
+	 */
+	public static function setUpBeforeClass() {
+		self::empty_the_store();
+	}
+
+	/**
+	 * Test get_meta_box_keys() returns empty array when store key is empty or does not start with 'meta_box.'
+	 */
+	public function test_returns_empty_array_when_store_key_is_empty_or_does_not_begin_with_meta_box() {
+		loadConfig( '', [] );
+		get_meta_box_keys();
+
+		$this->assertSame( [], get_meta_box_keys() );
```

Comment:
@rgadon107 The first 2 lines of code are not needed.

In addition to returning an empty array when the Config Store is empty, it should also return an empty array _when_ the store doesn't contain a config with `'meta_box.'`. How would you test for this?

1. Load a config into the store that doesn't have a meta box. For example, use the config in the test below but remove the 2 meta box configs from it. 
2. Then test that you get an empty array back.
3. Then empty the store when done.

Reply:
Done.